tests/framework/installer/ceph_manifests_v1.0.go

```diff
@@ -972,6 +972,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: ROOK_ENABLE_FLEX_DRIVER
```

Comment:
Is this needed to make tests pass?

Reply:
Yes, I believe it is.

Reply:
Actually, can you change this in `ceph_manifests.go` and revert the change in this file? This will only affect the upgrade test from 1.0 where that setting didn't exist.

Reply:
Oops, yes, done!